lib/alerts/components/AlertEditor.js

```diff
@@ -221,10 +220,16 @@ export default class AlertEditor extends Component<Props> {
                             : 'text-danger'
                         }
                         style={{fontWeight: 400}}>
-                        {titleCharactersRemaining}
+                        {titleCharacterCount}
                       </span>
                       <h5 style={{margin: '0px'}}>
                         <small>
+                          {titleCharacterCount > ALERT_TITLE_CHAR_LIMIT
+                            ? (
+                              <span className='text-danger'>
+                                {`WARNING:  Alert title longer than ${ALERT_TITLE_CHAR_LIMIT} characters may get truncated in some dissemination channels.  `}<br />
```

Comment:
There's no need to encapsulate this in a string/curly braces. That's for evaluating javascript, but if you have a bunch of string literals you want to render, you can just place them directly in the jsx.
```suggestion
                                WARNING:  Alert title longer than ${ALERT_TITLE_CHAR_LIMIT} characters may get truncated in some dissemination channels.<br />
```